accessible_api/AccessibilityAPIServer.py

```diff
@@ -0,0 +1,42 @@
+""" API for Accessible Requests """
+
+from SocketServer import TCPServer
+from argparse import ArgumentParser
+from AccessibilityRequestHandler import PlatformAccessibilityRequestHandler
+
+
+VERBOSE = False
+
+
+class AccessibilityAPIServer:
+    def __init__(self, port, platform=None, ip=""):
+        if VERBOSE:
+            print '.............SETTING UP SERVICE............'
+        handler = PlatformAccessibilityRequestHandler(platform)
```

Comment:
So semantically this is not a handler but a factory. This function returns a platform specific class depending on the arg or system string. Lets rename it into platform_accessibility_request_handler_factory (i little long but descriptive IMHO :), also functions in python have an underscore notation not camelcase).